XcodeServerSDK/Utils/JSON.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 public protocol JSONSerializable {
-    init?(json: NSDictionary)
+    init?(json: NSDictionary) throws
```

Comment:
We can slowly implement the `throws` on all the `classes` that implement `JSONSerializable` since it will be a big change because we'll have to `do`, `try`/`catch` and react accordingly